tests/FakeItEasy.IntegrationTests/FakingClassesTests.cs

```diff
@@ -12,10 +11,13 @@ public class FakingClassesTests
         [Test]
         public void Should_be_able_to_get_a_fake_value_of_uri_type()
         {
-            using (Fake.CreateScope(new NullFakeObjectContainer()))
-            {
-                A.Fake<Uri>();
-            }
+            // Arrange
+
+            // Act
+            var fake = A.Fake<Uri>();
+
+            // Assert
+            Fake.GetFakeManager(fake).Should().NotBeNull("because we should be able to create a fake Uri");

```

Comment:
Oh!
I didn't see "I'll send the PR" in my e-mail client.
I was making an issue, which I was going to take, as I had no objection to doing the work, now that we've agreed what the right work is. Will close my issue. Sorry/thanks.